client/jars/org.jboss.tools.rsp.client.cli/src/main/java/org/jboss/tools/rsp/client/bindings/ServerManagementClientImpl.java

```diff
@@ -7,18 +7,23 @@
  * Contributors: Red Hat, Inc.
  ******************************************************************************/
 package org.jboss.tools.rsp.client.bindings;
-/* --------------------------------------------------------------------------------------------
- * Licensed under the MIT License. See License.txt in the project root for license information.
- * ------------------------------------------------------------------------------------------ */
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.jboss.tools.rsp.api.ICapabilityKeys;
 import org.jboss.tools.rsp.api.RSPClient;
 import org.jboss.tools.rsp.api.RSPServer;
 import org.jboss.tools.rsp.api.ServerManagementAPIConstants;
+import org.jboss.tools.rsp.api.dao.CapabilitiesRequest;
+import org.jboss.tools.rsp.api.dao.CapabilitiesResponse;
 import org.jboss.tools.rsp.api.dao.DiscoveryPath;
 import org.jboss.tools.rsp.api.dao.ServerHandle;
 import org.jboss.tools.rsp.api.dao.ServerProcess;
 import org.jboss.tools.rsp.api.dao.ServerProcessOutput;
 import org.jboss.tools.rsp.api.dao.ServerStateChange;
+import org.jboss.tools.rsp.api.dao.StringPrompt;
 
 public class ServerManagementClientImpl implements RSPClient {
 	
```

Comment:
RSPServer instance var should be private and final I think:
```
private final RSPServer server
```

Reply:
made private, can't make final bc it's set during the initialization()